package.json

```diff
@@ -37,10 +37,11 @@
     ]
   },
   "dependencies": {
-    "@ember-template-lint/todo-utils": "^3.2.0",
+    "@ember-template-lint/todo-utils": "^4.1.0",
     "chalk": "^4.0.0",
     "ember-template-recast": "^5.0.1",
     "find-up": "^5.0.0",
+    "fs-extra": "^9.0.1",
```

Comment:
Hmm, where is this used?

Reply:
I think it was added during the time before we extracted the package.json/todo config code. I'll remove, since it's not used.

Reply:
Done.